frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -436,20 +406,20 @@ export function FieldValuesWidgetInner({
       >
         {isListMode && isLoading ? (
           <LoadingState />
-        ) : isListMode && hasListValues && multi ? (
+        ) : isListMode && !_.isEmpty(options) && multi ? (
```

Comment:
not clear what does it mean from business logic `!_.isEmpty(options)` - it worth having a well named variable for it as per `isListMode`

Reply:
Good point. I should use `const hasListValues = !_.isEmpty(options)`

Reply:
Decided to use `const hasOptions = !_.isEmpty(options)` instead, to be consistent with `options` elsewhere
cefeebeb8d5a49fa5a8cf71f92c5d4efc66dacf7